app/src/components/ui/organisms/Pomodoro/components.tsx

```diff
@@ -0,0 +1,35 @@
+import { useStore } from '@nanostores/react'
+
+import { handlePhase } from '@store/Pomodoro/actions'
+import { pomodoroStore } from '@store/store'
+import type { PhaseType } from '@store/types'
+
+import Button from '@atoms/Buttons/Button'
+
+import { BUTTON_TEXT } from './constants'
+import './styles.css'
+
+export const PhaseButtons = () => {
+  const { phase } = useStore(pomodoroStore.state)
+
+  const buttonPhases = ['pomodoro', 'break', 'longBreak'] as const
+
+  return (
+    <div className="pomodoro__buttons">
+      {buttonPhases.map((buttonPhase, index) => {
+        let styles = 'button '
+        styles = buttonPhase === phase ? 'button__chosenOption' : 'button__options'
+
+        return (
+          <Button
+            key={index}
+            handleClick={() => handlePhase(buttonPhase as PhaseType)}
```

Comment:
Avoid using `as` if it's not necessary.
Here it's not necessary as you can type `buttonPhase` when you declare it as `PhaseType`, and it will be picked up as `PhaseType` automatically here.